ceph_deploy/hosts/__init__.py

```diff
@@ -6,17 +6,17 @@
 """
 import logging
 from ceph_deploy import exc
-from ceph_deploy.hosts import debian, centos, fedora, suse, remotes
+from ceph_deploy.hosts import debian, centos, fedora, suse, remotes, rhel
 from ceph_deploy.connection import get_connection
 
 logger = logging.getLogger()
 
 
-def get(hostname, username=None, fallback=None, detect_sudo=True):
+def get(hostname, args, fallback=None, detect_sudo=True):
```

Comment:
I can definitely do that instead.  The initial reason was that I wanted access to the loaded cephdeploy.conf file, which is stored in args.cd_conf.  But then I didn't actually need that, because when the config file is parsed, all the key/value pairs are adde to args as attributes (first time I've had to use that part of the code).  So I changed the way I was doing it halfway through.

So yes, I could just pass use use_rhceph as  a kwarg instead.